ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -169,115 +147,57 @@ const secondaryPanel = panel => ({
       annotations filter.
     </span>
   ),
-})
-const ANNOTATION_SECONDARY_PANEL_MAP = {
-  ...secondaryPanel(ANNOTATION_PANEL),
   fieldLayout: annotationFieldLayout(
     [SV_GROUPS_NO_NEW, HIGH_IMPACT_GROUPS, MODERATE_IMPACT_GROUPS, CODING_IMPACT_GROUPS],
   ),
-  [DATASET_TYPE_SV_CALLS]: {
-    ...secondaryPanel(ANNOTATION_PANEL_MAP[DATASET_TYPE_SV_CALLS]),
-    fieldLayout: annotationFieldLayout([SV_GROUPS_NO_NEW], true),
-  },
-  [DATASET_TYPE_VARIANT_CALLS]: {
-    ...secondaryPanel(ANNOTATION_PANEL_MAP[DATASET_TYPE_VARIANT_CALLS]),
-    fieldLayout: annotationFieldLayout([HIGH_IMPACT_GROUPS, MODERATE_IMPACT_GROUPS, CODING_IMPACT_GROUPS]),
-  },
-}
-
-const svCallsetChange = (onChange, initialValues) => val => onChange(
-  { ...initialValues, [SV_CALLSET_FREQUENCY]: val },
-)
-
-const SVFrequecyHeaderFilter = ({ value, onChange }) => (
-  <Form.Group inline>
-    <AfFilter
-      value={value[SV_CALLSET_FREQUENCY]}
-      onChange={svCallsetChange(onChange, value)}
-      inline
-      label="Callset"
-      width={16}
-    />
-  </Form.Group>
-)
-
-SVFrequecyHeaderFilter.propTypes = {
-  value: PropTypes.object,
-  onChange: PropTypes.func,
-}
-
-const SV_QS_FILTER_FIELD = {
```

Comment:
these were moved to the shared constants, as we want the default panels to have all dataset types